chunjun-connectors/chunjun-connector-jdbc-base/src/main/java/com/dtstack/chunjun/connector/jdbc/util/JdbcUtil.java

```diff
@@ -497,4 +498,78 @@ public static void resetSchemaAndTable(JdbcConf jdbcConf, String leftQuote, Stri
                     jdbcConf.getTable());
         }
     }
+
+    public static Pair<List<String>, List<String>> checkAndModifyColumnInfo(
+            JdbcConf jdbcConf,
+            Pair<List<String>, List<String>> tableMetaData,
+            String constantType) {
+        List<String> metaColumnName = tableMetaData.getLeft();
+        List<String> metaColumnType = tableMetaData.getRight();
+        int size = metaColumnName.size();
+        List<String> columnNameList = new ArrayList<>(size);
+        List<String> columnTypeList = new ArrayList<>(size);
+        List<FieldConf> column = jdbcConf.getColumn();
+        if (StringUtils.isNotBlank(jdbcConf.getCustomSql())) {
```

Comment:
There is no obvious relation between the 'if' and 'else', i think use different function is better.